src/stories/StyledModal.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { PropsWithChildren } from 'react';
+import StyledButton from './StyledButton';
+
+export enum StyledModalColors {
+  GRAY = 'GRAY',
+  WHITE = 'WHITE',
+  DFXGRADIENT = 'DFXGRADIENT',
+}
+
+interface StyledModalProps extends PropsWithChildren {
+  onClose: (showModal: boolean) => any;
+  closeWithX?: boolean;
```

Comment:
`closeWithX` is unused

Reply:
yeah, there's more work on modal ahead ... it will be used :)

Reply:
needed to build the iconButtons first